pegasus/sites.v3/code.org/public/project-ideas.md.partial

```diff
@@ -95,7 +95,7 @@ students. Programming in a brand new environment can be challenging and
 intimidating. We recommend giving yourself sufficient time to learn to use
 them. Don't give up! Below you can find an overview of each tool and also how
 to practice using it. For more information about full course offerings, check
-out [CS Discoveries](https://code.org/educate/csd)for middle school or [CS
+out [CS Discoveries](https://code.org/educate/csd) for middle school or [CS
```

Comment:
one last nit: remove this so the change stays scoped to relevant files (good catch though! I can forward this to our team to fix this in the relevant system)